Classes/Controller/BackendServiceController.php

```diff
@@ -545,4 +552,18 @@ function ($envelope) {
 
         return json_encode($result);
     }
+
+    /**
+     * Generates a new uri path segment for the given node and title
+     *
+     * @param NodeInterface $contextNode
+     * @param string $text
+     * @return string
+     * @throws \Neos\Neos\Exception
+     */
+    public function transliteratePathSegmentAction(NodeInterface $contextNode, string $text): string
+    {
+        $slug = $this->nodeUriPathSegmentGenerator->generateUriPathSegment($contextNode, $text);
+        return json_encode($slug);
```

Comment:
Use the JSON view feature like this:

```suggestion
        $this->view->assign('value', $slug);
```